jni/nnstreamer.mk

```diff
@@ -25,6 +25,10 @@ NNSTREAMER_COMMON_SRCS := \
     $(NNSTREAMER_GST_HOME)/nnstreamer_subplugin.c \
     $(NNSTREAMER_GST_HOME)/tensor_common.c
 
+ifneq ($(NNSTREAMER_API_OPTION),single)
```

Comment:
how about adding 'tensor_common_pipeline.c' in NNSTREAMER_PLUGINS_SRCS?

Reply:
Moved to NNSTREAMER_PLUGINS_SRCS along with a few comments to avoid confusion.